api/app/models/spree/webhooks/event.rb

```diff
@@ -0,0 +1,7 @@
+module Spree
+  module Webhooks
+    class Event < Spree::Base
```

Comment:
@sebastian-palma so what will this model hold? specific event logged in the DB?

Reply:
It should contain the specific events for every model that triggers a webhook (e.g order.create, order.cancel, etc), but as I see, such events should be inferred from the model name and the method being executed. I have to think again how to model this and get this done.